doc/installation.rst

```diff
@@ -11,12 +13,14 @@ Thanks!
 
 .. _issue tracker: https://github.com/spyder-ide/spyder/issues
 
-.. important::
+.. note::
 
-   Spyder now offers :ref:`standalone installers<standalone_installers_ref>` for Windows and macOS, making it easier to get up and running with the application without having to download Anaconda or manually install it in your existing environment.
-   While we still support Anaconda, we recommend this install method on those platforms to avoid most problems with package conflicts and other issues.
+   For most users on Windows and macOS, we recommend our :ref:`install-standalone` as the most straightforward and robust option to obtain Spyder.
+   For users on Linux, needing third-party Spyder plugins or Variable Explorer compatibility with custom-installed packages, capabilities which the standalone installers currently do not provide, we advise using a :ref:`install-conda`.
```

Comment:
```suggestion
   For users needing Linux support, third-party Spyder plugins or Variable Explorer compatibility with custom-installed packages—all capabilities which the standalone installers currently do not yet provide—we advise using a :ref:`install-conda`.
```

Instead of repeating users again as well as a doubled-up "or", I rewrote the sentence to make it clearer and simpler and avoid the need for that.